dbbackup/management/commands/dbrestore.py

```diff
@@ -77,7 +77,8 @@ def restore_backup(self):
         if not self.filepath:
             self.log("  Finding latest backup", 1)
             filepaths = self.storage.list_directory()
-            filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
+            # TODO: It is a bad filter
+            # filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
```

Comment:
Yep, I maybe write this comment a bit too fast,
But imagine I have ``.gz`` backups, this filter includes:
- Compressed backup of  `mediabackup`
- Other GZ files

Before and after this commit, the filter is not efficient.
We should be able to find which are db-backups and find last backup by a real meta data, not the last of list.

Do you want me to uncomment ? I think is better to talk about it in issue #64, Django storage API has: `create_time` attribute (https://docs.djangoproject.com/en/1.8/ref/files/storage/#django.core.files.storage.Storage.created_time) for this task.